ansible/roles/openshift_node_config/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+# tasks file for openshift_node_config
+- name: Edit OpenShift node config
+  lineinfile:
+    dest: /etc/sysconfig/atomic-openshift-node
+    line: "{{item.key }}={{ item.value }}"
```

Comment:
likely want a regex value passed so that we can catch the case where we want 'VAR=previous' set to 'VAR=new'

regex: "^{{ item.key }}="